SPECS/xcp-ng-release.spec

```diff
@@ -589,6 +589,9 @@ then
     systemctl reenable rsyslog.service
 fi
 
+# we are shipping a file in depmod.d/, ensure it is used
+/sbin/depmod -a || true
```

Comment:
It would be good to redirect stdout and stderr in order to avoid displaying error messages in the installer logs when depmod is not present yet.